modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -9,7 +9,20 @@
 
 To create CSI-provisioned persistent volumes that mount to these supported storage assets, {product-title} installs the necessary CSI driver Operator, the CSI driver, and the required storage class by default. For more details about the default namespace of the Operator and driver, see the documentation for the specific CSI Driver Operator.
 
-The following table describes the CSI drivers that are installed with {product-title} and which CSI features they support, such as volume snapshots, cloning, and resize.
+ifndef::openshift-rosa[]
+[IMPORTANT]
+====
+The AWS EFS and GCP Filestore CSI drivers are not installed by default, and must be installed manually. For instructions on installing the AWS EFS CSI driver, see link:https://docs.aws.amazon.com/efs/latest/ug/whatisefs.html[Amazon Elastic File System user guide]. For instructions on installing the GCP Filestore CSI driver, see link:https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/filestore-csi-driver[Access Filestore instances with the Filestore CSI driver].
```

Comment:
We have the procedure to install AWS-EFS CSI Driver here,
https://docs.openshift.com/rosa/storage/container_storage_interface/osd-persistent-storage-aws-efs-csi.html, I think we'd better to refer to ours, the giving link doesn't help install on Openshift-ROSA cluster.
Also I think we could remove GCP Filestore CSI drivers, usually we will not install it in RORA.

Reply:
Hello @duanwei33,
   Thanks for taking a look at this PR. To clarify, the changes I made will only appear on the OSD docs b/c they are OSD specific per the original JIRA ticket description https://issues.redhat.com/browse/OSDOCS-7407.
ROSA docs will not be changed in anyway. 
Given that, do you have any other changes you feel need to be made before approving from a QE standpoint? Thanks!

Reply:
Sorry I misunderstood, it IS for OSD.

My only comment is when introducing the AWS EFS CSI driver /  GCP Filestore CSI drivers, we could refer to our install doc which is present in OSD/OCP doc already.
It could be included in next doc jira/PR.

Reply:
Updated the links to point to our docs per suggestion.